addons/l10n_lv/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,557 @@
+              'repartition_type': 'base',
+              'tag_ids': [ref('VAT_51SG_tag')],
+            }),
+            (0,0, {
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+              'tag_ids': [ref('VAT_51PG_tag')],
+            }),
+            (0,0, {
+              'factor_percent': -100,
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+            }),
+        ]" />
+    <field name="refund_repartition_line_ids"
+      eval="[(5, 0, 0),
+            (0,0, {
+              'repartition_type': 'base',
+              'tag_ids': [ref('VAT_51SG_tag')],
+            }),
+            (0,0, {
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+              'tag_ids': [ref('VAT_51PG_tag')],
+            }),
+              (0,0, {
+              'factor_percent': -100,
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+            }),
+        ]" />
+  </record>
+
+  <!-- 5% VAT, VAT report R511 + PVN 1.2. -->
+  <record id="account_tax_template_sales_VAT_51112SG_tag" model="account.tax.template">
+    <field name="chart_template_id" ref="account_chart_template_latvia" />
+    <field name="name">Purchase 5% EU</field>
+    <field name="description">5% VAT</field>
+    <field name="type_tax_use">purchase</field>
+    <field name="amount">5.0</field>
+    <field name="tax_group_id" ref="tax_group_vat_5" />
+    <field name="amount_type">percent</field>
+    <field name="invoice_repartition_line_ids"
+      eval="[(5, 0, 0),
+            (0,0, {
+              'repartition_type': 'base',
+              'tag_ids': [ref('VAT_51112SG_tag')],
+            }),
+            (0,0, {
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+              'tag_ids': [ref('VAT_51112PG_tag')],
+            }),
+            (0,0, {
+              'factor_percent': -100,
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+            }),
+        ]" />
+    <field name="refund_repartition_line_ids"
+      eval="[(5, 0, 0),
+            (0,0, {
+              'repartition_type': 'base',
+              'tag_ids': [ref('VAT_51112SG_tag')],
+            }),
+            (0,0, {
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+              'tag_ids': [ref('VAT_51112PG_tag')],
+            }),
+              (0,0, {
+              'factor_percent': -100,
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+            }),
+        ]" />
+  </record>
+
+  <!--  VAT, VAT report PVN2 services-->
+  <record id="account_tax_template_sales_VAT_2P_tag" model="account.tax.template">
+    <field name="chart_template_id" ref="account_chart_template_latvia" />
+    <field name="name">VAT2 Services</field>
+    <field name="description">W/O VAT (EU)</field>
+    <field name="type_tax_use">sale</field>
```

Comment:
Not a big deal, but could we please define all sales taxes first, then the purchase taxes?